### PR TITLE
add _66.java solution2

### PR DESCRIPTION
**Pull request by WillShen7789**

### CODE REVIEW

**Comment by yaodehaung** [comment]
@WillShen7789 great

File: src/test/java/com/fishercoder/_66Test.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.fishercoder;
 
+import com.fishercoder.solutions._1005;
```
Comment (fishercoder1534) [inline src/test/java/com/fishercoder/_66Test.java:3]: why here?

File: src/test/java/com/fishercoder/_66Test.java [hunk 2]
```diff
@@ -8,28 +9,44 @@
...
     public static void setup() {
         solution1 = new _66.Solution1();
+        solution2 = new _66.Solution2();
     }
 
     @Test
     public void test1() {
         digits = new int[]{9, 9, 9, 9};
         assertArrayEquals(new int[]{1, 0, 0, 0, 0}, solution1.plusOne(digits));
+        digits = new int[]{9, 9, 9, 9};
```
Comment (fishercoder1534) [inline src/test/java/com/fishercoder/_66Test.java:25]: Please write new tests for `solution2` so each test is responsible for only one solution. Thanks

**Review by fishercoder1534 — CHANGES_REQUESTED** [review]

File: src/test/java/com/fishercoder/_66Test.java (hunk 1 above)
Comment (WillShen7789) [reply src/test/java/com/fishercoder/_66Test.java:3]: My mistake.

**Comment by WillShen7789** [comment]
1.write new tests for solution2
2.remove import

**Review by fishercoder1534 — APPROVED** [review]

**Pull request merged**